Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
This topic will show up after other existing topics, according to `.pages`. That seems fine, just wanted to confirm.

Reply:
I don't think ordering is strongly important here, but happy to reorder if needed.